modules/linear_boltzmann_solvers/a_lbs_solver/lbs_solver.cc

```diff
@@ -980,7 +980,7 @@ LBSSolver::InitializeMaterials()
                               << "energy groups than called for in the simulation. "
                               << "Source will be ignored.";
         }
-        else
+        else if (not options_.adjoint)
         {
```

Comment:
This could probably be rewritten to be more clear, but the logic is correct. We only use material sources in forward solves at this point, so the else would be to do nothing.